README.md

```diff
@@ -64,107 +64,82 @@ module Client =
   
   ## Query
   
-  ### Query Configuration
-  **QueryConfig.re**
+  **MyComponent.re**
   ```reason
-  /* Create a query with the `graphql-tag` */
-  
-  let query = [@bs] gql({|
-    query getUser {
-      name
+  /* Create a GraphQL Query by using the graphql_ppx */ 
+  module PokemonQuery = [%graphql {|
+    query getPokemon($name: String!){
+        pokemon(name: $name) {
+            name
+        }
     }
-  |});  
-  
-  /* Describe the result type */
```

Comment:
Ah yeah, another question. Previously you had to define the shape of the response to get a typed object. This is now done implicitly by the `parse` function if I'm not mistaken, right?

Reply:
Not exactly. graphql_ppx, will generate a `parse` function that has type `Js.Json.t => GeneratedResponseType`
So it generated a `GeneratedResponseType` and a function to parse a `Js.Json.t` to your response type.

Reply:
Thanks! I guess I have to try it out myself to get familiar with it.